doc/test-framework/scorecard.md

```diff
@@ -325,11 +325,157 @@ Example of a valid JSON output:
+          ```sh
+          $ kubectl apply -f scorecard-secret.yaml
+          ```
+    1. Insert a volume referring to the `Secret` into the operator's `Deployment`:
+          ```yaml
+          spec:
+            install:
+              spec:
+                deployments:
+                - name: memcached-operator
+                  spec:
+                    ...
+                    template:
+                      ...
+                      spec:
+                        containers:
+                        ...
+                        volumes:
+                        # scorecard kubeconfig volume
+                        - name: scorecard-kubeconfig
+                          secret:
+                            secretName: scorecard-kubeconfig
+                            items:
+                            - key: kubeconfig
+                              path: config
+          ```
+1. Insert a volume mount and `KUBECONFIG` environment variable into each container in your operator's `Deployment`:
+    ```yaml
+    spec:
+      install:
+        spec:
+          deployments:
+          - name: memcached-operator
+            spec:
+              ...
+              template:
+                ...
+                spec:
+                  containers:
+                  - name: container1
+                    ...
+                    volumeMounts:
+                    # scorecard kubeconfig volume mount
+                    - name: scorecard-kubeconfig
+                      mountPath: /scorecard-secret
+                    env:
+                      # scorecard kubeconfig env
+                    - name: KUBECONFIG
+                      value: /scorecard-secret/config
+                  - name: container2
+                    # Do the same for this and all other containers.
+                    ...
+    ```
+1. Insert the scorecard proxy container into the operator's `Deployment`:
+    ```yaml
+    spec:
+      install:
+        spec:
+          deployments:
+          - name: memcached-operator
+            spec:
+              ...
+              template:
+                ...
+                spec:
+                  containers:
+                  ...
+                  # scorecard proxy container
+                  - name: scorecard-proxy
+                    command:
+                    - scorecard-proxy
+                    env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          apiVersion: v1
+                          fieldPath: metadata.namespace
+                    image: quay.io/operator-framework/scorecard-proxy:master
+                    imagePullPolicy: Always
+                    ports:
+                    - name: proxy
+                      containerPort: 8889
+    ```
+
+Alternatively, the [community-operators][community-operators] repo has several bash functions that can perform these operations for you:
```

Comment:
Is the following commands,  from the community-operators, doing the same tests that will be done for the above steps? If not, I'd recommend replacing `Alternatively` by something as `Also,`.